examples/connect-app-teleop-react/frontend/env.ts

```diff
@@ -0,0 +1,56 @@
+export interface BuildEnvironment {
+  baseUri: string;
+  orgId: string;
+  isDev: boolean;
+  auth:
+  | { case: 'api_key'; apiKeyId: string; apiKeySecret: string }
+  | { case: 'third_party'; clientId: string };
+}
+
+let env: BuildEnvironment | undefined;
+
+const throwNotSet = (envVarKey: string): never => {
+  throw new Error(`${envVarKey} not set`);
+};
```

Comment:
Is there any way we could make these build-time errors instead? Nice to ditch busy-work client-side code out of the example

Reply:
yes